pkg/cmd/list/rule_engine/rule_engine.go

```diff
@@ -100,21 +95,15 @@ func PrintTable(cmd *cobra.Command, f *cmdutil.Factory, opts *contracts.ListOpti
 	columnFmt := color.New(color.FgGreen).SprintfFunc()
 	tbl.WithHeaderFormatter(headerFmt).WithFirstColumnFormatter(columnFmt)
 
-	if cmd.Flags().Changed("details") {
-		for _, v := range rules.Results {
-			tbl.AddRow(v.Id, v.Name, v.Order, v.Phase, v.IsActive)
-		}
-	} else {
-		for _, v := range rules.Results {
-			tbl.AddRow(v.Id, v.Name)
-		}
+	for _, v := range rules.Results {
```

Comment:
why remove details?

Reply:
Remember what I said about it only showing the active ones?